xmodule/library_content_block.py

```diff
@@ -135,17 +141,6 @@ def completion_mode(cls):  # pylint: disable=no-self-argument
         display_name=_("Library Version"),
         scope=Scope.settings,
     )
-    mode = String(
-        display_name=_("Mode"),
-        help=_("Determines how content is drawn from the library"),
-        default="random",
-        values=[
-            {"display_name": _("Choose n at random"), "value": "random"}
-            # Future addition: Choose a new random set of n every time the student refreshes the block, for self tests
-            # Future addition: manually selected blocks
-        ],
-        scope=Scope.settings,
-    )
```

Comment:
> Am I correct in thinking that this is okay to remove because v1 content libraries always pulled randomly

yup!

> this was only ever added in here as an affordance to v2 content libraries?

It was actually added back in the v1 library days, before non-random library reference was cut from that project's scope.